newsroom/agenda/__init__.py

```diff
@@ -5,7 +5,6 @@
 from newsroom.utils import url_for_agenda
 from .agenda import AgendaResource, AgendaService, aggregations, PRIVATE_FIELDS
 from newsroom.search.config import init_nested_aggregation
-from .featured import FeaturedResource, FeaturedService
```

Comment:
Is `newsroom.agenda.featured` module use anywhere in the code now, if not it would be good to remove that file